src/http-client/paths.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * Readonly object representing the paths of the Fauna API to be used
+ * with HTTP clients.
+ */
+export const FaunaAPI = {
+  QUERY: "/query/1",
+  STREAM: "/stream/1",
+} as const;
+
+export type SupportedFaunaAPIs = (typeof FaunaAPI)[keyof typeof FaunaAPI];
```

Comment:
Nit question: should we name these `...Paths`? `FaunaAPIPaths` and `SupportedFaunaAPIPaths`?